<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCliMainWind</class>
 <widget class="QMainWindow" name="QCliMainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCliMainWind</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>518</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ListenAddress">
         <property name="text">
          <string>服务器地址:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="IPcomboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ListenPort">
         <property name="text">
          <string>端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="PortspinBox">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1200</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="MessageBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>490</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>490</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="MessageLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Message:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="ClearMessageButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>清空消息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SendMessageButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>发送信息</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="MessageEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="StateText">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="MemberView">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actConnect"/>
   <addaction name="actDisconnect"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actCloseWindow"/>
  </widget>
  <action name="actConnect">
   <property name="icon">
    <iconset>
     <normalon>icon/连接.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>连接到服务器</string>
   </property>
   <property name="toolTip">
    <string>连接到服务器</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="icon">
    <iconset>
     <normalon>icon/断开连接.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
   <property name="toolTip">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actCloseWindow">
   <property name="icon">
    <iconset>
     <normalon>icon/退出.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset>
     <normalon>icon/清空.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="toolTip">
    <string>清空文本框</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actCloseWindow</sender>
   <signal>triggered()</signal>
   <receiver>QCliMainWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>MessageBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearMessageButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
